modeling/model_utils/model_pipeline_io.py

```diff
@@ -11,7 +13,22 @@ def get_para() -> dict:
     return paras
 
 
-def read_train_file() -> pd.DataFrame:
+def read_train_file(filename: str) -> pd.DataFrame:
     paras = get_para()
-    df = pd.read_csv(paras["path_to_clean_train_file"], index_col="company_id")
+    filepath = os.path.join(paras["path_to_output_dir"], filename)
+    df = pd.read_csv(filepath, index_col="company_id")
     return df
+
+
+def get_training_set(train_set_name: list):
+    train_data = pd.concat(
+        [read_train_file(name).iloc[:, :-1] for name in train_set_name], axis=1,
+    )
+    train_data_with_target = pd.concat(
+        [train_data, read_train_file(train_set_name[0]).iloc[:, -1]], axis=1
+    )
+
+    train_data_with_target = drop_duplicate.remove_null_and_duplicate_rows(
+        train_data_with_target
+    )
```

Comment:
same question: are we removing null and duplicate values in this part rather than in DE pipeline?